### DISCUSSION TITLE
I created a pod yesterday, it's there, but podman desktop says "No Pods"?

### DISCUSSION THREAD

**Discussion by angelotrivelli** [podman-desktop/podman-desktop#1427] [Q&A] [answered]
Hi, sorry, I am new to podman. 

I followed a tutorial yesterday to create a pod that runs postgres-SQL and pgAdmin. It was straight forward and everything seem to work fine. 

Today, upon launching podman, the pod I created doesn't appear any more in Podman Desktop.

But when I run...
```
    PS C:\Users\user> podman pod ps
    POD ID        NAME         STATUS      CREATED       INFRA ID      # OF CONTAINERS
    af11eaf3be72  postgre-sql  Created     27 hours ago  703c042a3f27  3
```
I see that ps lists my pod, and I can even run it and it works fine, I can navigate on the browser to pgadmin and work with the containerized postgres.

```
    PS C:\Users\user> podman pod restart af1
    af11eaf3be72a3042b26bc985b572eec5198731da59bb42a9d589f602824d752
    PS C:\Users\user> podman pod ps
    POD ID        NAME         STATUS      CREATED       INFRA ID      # OF CONTAINERS
    af11eaf3be72  postgre-sql  Running     27 hours ago  703c042a3f27  3
    PS C:\Users\user>
```

When I look at Podman Desktop, however...
[https://imgur.com/a/uAMbW5c](https://imgur.com/a/uAMbW5c)
It says "No Pods" (as well as "No Containers" and "No Images").

I am running on windows 10 with Podman v4.4.1 (and whatever is yesterday's version of Podman Desktop).  

The only clue I see that looks like something is wrong is on the Dashboard. It says:
`"Docker Socket Compatibility: Podman is not emulating the default Docker socket path: //./pipe/docker_engine. Docker specific tools may not work."`

But I am not running docker. I did everything with podman. So what is a "docker specific tool"? 

I see there's some short hand advice for experts on how to resolve this but I am not able to follow. Can someone explain, in simple terms, what's going on and how to fix this so that Podman Desktop actually displays the pods/container/images which are running in Podman?

Thanks!

**Comment by benoitf** [comment, marked as answer]
Hello @angelotrivelli is that with a restart of Podman Desktop you saw your pods ?

If you go to the menu View/Toggle Developer tools and then Console do you see some errors ?
(menu canbe displayed by pressing the left <kbd>ALT<kbd> key

Could you say details about your machine ? `podman machine inspect`

**Reply by angelotrivelli** [reply]
Thank you!

I quit and then restarted Podman Desktop and upon coming back I could see the pod, containers, images and volumes. It even works now across reboots (no need to restart podman desktop). I did install podman first and then Podman Desktop. Maybe that had something to do with it.

Also, I have a message on the dashboard saying the following:
`Docker Socket Compatibility: Podman is not emulating the default Docker socket path: //./pipe/docker_engine. Docker-specific tools may not work.
`
I don't know how to interpret that message. Is it a concern? If I do everything with podman cli and podman desktop, there's no problem, right?

I had been worried that I wasn't seeing the menu-bar, and yes, the alt-key displays it. Thanks again! I would not have found that myself! Maybe it would be better to have the menu bar enabled by default upon installation? If I didn't know the key-binding, then many others will not either,

Finally, here is my `podman machine inspect` output:
```
PS C:\Users\AngeloTrivelli> podman machine inspect
[
     {
          "ConfigPath": {
               "Path": "C:\\Users\\AngeloTrivelli\\.config\\containers\\podman\\machine\\wsl\\podman-machine-default.json"
          },
          "ConnectionInfo": {
               "PodmanSocket": null
          },
          "Created": "2023-02-09T11:21:39.4334586-05:00",
          "Image": {
               "IgnitionFilePath": {
                    "Path": ""
               },
               "ImageStream": "35",
               "ImagePath": {
                    "Path": "C:\\Users\\AngeloTrivelli\\.local\\share\\containers\\podman\\machine\\wsl\\podman-machine-default_fedora-podman-amd64-v36.0.123.tar"
               }
          },
          "LastUp": "2023-02-13T08:06:31.6808789-05:00",
          "Name": "podman-machine-default",
          "Resources": {
               "CPUs": 12,
               "DiskSize": 2557476864,
               "Memory": 665358336
          },
          "SSHConfig": {
               "IdentityPath": "C:\\Users\\AngeloTrivelli\\.ssh\\podman-machine-default",
               "Port": 51285,
               "RemoteUsername": "user"
          },
          "State": "running"
     }
]
```

**Comment by benoitf** [comment]
thanks for the follow-up

> I quit and then restarted Podman Desktop and upon coming back I could see the pod, containers, images and volumes. It even works now across reboots (no need to restart podman desktop). I did install podman first and then Podman Desktop. Maybe that had something to do with it.

maybe Podman Desktop lost the connection to the socket (and then the events so it was not notified)

> Also, I have a message on the dashboard saying the following:
Docker Socket Compatibility: Podman is not emulating the default Docker socket path: //./pipe/docker_engine. Docker-specific tools may not work. 
I don't know how to interpret that message. Is it a concern? If I do everything with podman cli and podman desktop, there's no problem, right?


here it's more that if you use tools that expect to have Docker installed that it may not work (for example some IDE tooling, etc)
but if you use only podman cli and podman, no worries


> I had been worried that I wasn't seeing the menu-bar, and yes, the alt-key displays it. Thanks again! I would not have found that myself! Maybe it would be better to have the menu bar enabled by default upon installation? If I didn't know the key-binding, then many others will not either,

here I think maybe we need a property in the settings as some people created PR to hide it to have a better display, some to make it visible all the time so I'm 🙃
